feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableTab.kt

```diff
@@ -179,8 +181,11 @@ class TimetableTabState(
 }
 
 private val minTabHeight = 32.dp
+private val baselineTabHeight = 56.dp
 private val maxTabRowHeight = 84.dp
 private val minTabRowHeight = 56.dp
+private val tabIndicatorHorizontalSpacing = 8.dp
+private val tabRowHorizontalSpacing = (maxTabRowHeight - baselineTabHeight) / 2 - tabIndicatorHorizontalSpacing
```

Comment:
May I ask why we used `maxTabRowHeight - baselineTabHeight` here? Is it a size for the corner round?

Reply:
Let me explain :raising_hand_man:

Tabs are already vertically center aligned of the parent container element (`TabRow`), so I thought the horizontal spacing size should be the same as the vertical ones.

However, I noticed my mistake after this PR getting merged. (Sorry!) I checked the UI design spec (Figma) and it says a bit different layout constraints specified to Tabs and TabRow. They are not vertically aligned and the bottom padding amount is less than other directions.

I'm going to create a PR to match the implementation with the UI design spec later.

![g112452](https://github.com/DroidKaigi/conference-app-2023/assets/2552365/3e96a888-f82c-48b7-825f-1f9b42ace0a3)